libraries/inverse/inverse_global.h

```diff
@@ -42,6 +42,7 @@
 //=============================================================================================================
 
 #include <QtCore/qglobal.h>
+#include <utils/mnetracer.h>
```

Comment:
why is this include in the inverse_global.h file? This way it will be included in a lot of headers files even if we do not use the mne tracer functionality. We should only include what we actually use.